### ISSUE TITLE
Apply to BOLD.gov 

### ISSUE THREAD

**Issue by lpgoldstein** [ONRR/nrrd#1770]
Each year at BOLD GOV, Government Executive and Nextgov team up to explore the powerful combination of technology, culture, and people that enables government to deliver on its essential mission.

https://www.govexec.com/f/2021-bold-gov-nomination-form/
 
Do you have colleagues that are transforming the way government operates and provides services? Nominate them to present at BOLD GOV, a virtual celebration of innovation in government! 

A panel of Government Executive and Nextgov editors will review the nominations and select the top nominees to present at this year's BOLD GOV. Here are a few examples of past presentations:

How Tablets Kept the Asylum Program Safe: Creative Digital Responses to the COVID Emergency at USCIS 
Innovative Methodology for Assessing Racial Disparities in Promotions
Autonomous Vehicle Pilot - Yellowstone National Park
Inclusive Staff Wellness and Resilience Training Program
Enabling Enterprise Data Sharing at HHS

Nominate yourself or a colleague who's doing exceptional work. Nominations across all agencies, seniority levels, and scope will be considered. The nomination deadline is Wednesday, October 20 at 11:59 PM ET. 

 

If you have any questions, please contact Kayla at kmccloud@govexec.com.

**Comment by cthomasONRR** [comment]
@lpgoldstein here's my write-up for the 508 presentation. Please feel free to modify!

**What is the proposed theme or title of the presentation?**
From 5,000 to 0: Prioritizing Documents for 508-Compliant Remediation 

**Please give us a four sentence description of what this presentation would showcase.** 
When the Director of the Office of Natural Resources Revenue (ONRR) decided to ensure all documents on onrr.gov are Section 508 compliant in July of 2020, we thought it would be a reasonable project. After ONRR’s director started the push to make documents compliant, “508 compliance” became a “buzzword” at our agency. This presentation focuses on our plan to correct all the documents currently on the website, and our plan going forward for any new documents requested to be uploaded on the website. This is our story of how we remediated the 5,000 documents on onrr.gov to be 508-compliant! 

**There is a lot of innovation going on in government today. What makes this program or project stand out? What is bold about it?** 
Accessibility is the foundation for inclusion, diversity, and equity within the federal government. As agencies work toward making their websites and documents 508-compliant, they will encounter many obstacles during this often daunting task. Our presentation outlines tangible steps that other agencies can apply and follow to ensure 508-compliance. We hope that our experiences will inspire and motivate other agencies to remediate all of their documents, no matter how old! In addition, we discuss the use of GitHub for 508-compliance tasks, which is fedRAMP authorized and free!

**Comment by cthomasONRR** [comment]
@lpgoldstein Yours looks good! Should I write more for mine??

**Comment by lpgoldstein** [comment]
@cthomasONRR I think yours looks good too - it has all of the keywords but isn't too wordy. I'm worried mine is too long. @mcharg or @Maroyafaied do you want to review before we submit?

**Comment by lpgoldstein** [comment]
@cthomasONRR I did the same thing.
